<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>408</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/nmap-logo-small.png</normaloff>../resources/nmap-logo-small.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:ArchitectsDaughter;font-size:14px;font-weight:normal;
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,1" columnstretch="1,1" rowminimumheight="1,1,1,1" columnminimumwidth="200,200">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_cancel">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="theme_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Theme</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QComboBox" name="theme_select">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select theme</string>
        </property>
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="splash_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Show Splash Animation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <item>
       <widget class="QRadioButton" name="show_anim">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="search_on_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Set search on key pressed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QRadioButton" name="search_on">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_accept">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="lang_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Language</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="lang_es">
        <property name="toolTip">
         <string>Set Spanish like default language</string>
        </property>
        <property name="text">
         <string>Spanish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lang_en">
        <property name="toolTip">
         <string>Set English like default language</string>
        </property>
        <property name="text">
         <string>English</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="search_by_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Search Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>35</number>
      </property>
      <item>
       <widget class="QRadioButton" name="search_by_name">
        <property name="toolTip">
         <string>Set search by name</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="search_by_author">
        <property name="toolTip">
         <string>Set search by author</string>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="search_by_category">
        <property name="toolTip">
         <string>Set search by category</string>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_accept</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_on</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>update_text()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_anim</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>update_anim_text()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>cancel()</slot>
  <slot>update_text()</slot>
  <slot>update_anim_text()</slot>
 </slots>
</ui>
